### DISCUSSION TITLE
Paging doesn't work when Renovate fetches docker tag list from a private registry

### DISCUSSION THREAD

**Discussion by catostrophe** [renovatebot/renovate#18448] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.209.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

14.10.5-ee

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

We have a corporate Nexus with a proxy to dockerhub. The Renovate image in `.gitlab-ci.yml`  is referred to as `nexus.mycompany.io/renovate/renovate:32.209.0-slim`.

A few weeks ago Renovate stopped upgrading itself.

I checked the logs. It fetches the list of tags from "https://nexus.mycompany.io/v2/renovate/renovate/tags/list?n=10000".

The json returned by this query:
```json
{"name":"renovate/renovate","tags":["0","0-slim","0.0","0.0-slim","0.0.1","0.0.1-slim",
"...thousands of tags...",
"32.208","32.208-slim","32.208.0","32.208.0-slim","32.208.1","32.208.1-slim",
"32.208.2","32.208.2-slim","32.208.3","32.208.3-slim","32.209","32.209-slim",
"32.209.0","32.209.0-slim"]}
```

But there are no link headers in the response, so paging doesn't work. And the latest version seen by Renovate is "32.209.0-slim".

Is it an issue with our particular Nexus proxy? Or doesn't Renovate support paging for Nexus at all?

 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
if nexus isn't sending paging headers, it's a nexus issue, not a renovate bug

**Reply by catostrophe** [reply]
I am not sure whether it's my Nexus not sending headers or any Nexus. In the latter case, Renovate should have a fix to support Nexus.

**Reply by catostrophe** [reply]
As a workaround, it's ok if I can amend limit 10000 via config or env.

**Comment by catostrophe** [comment, marked as answer]
It's a nexus issue: https://issues.sonatype.org/browse/NEXUS-26037

**Reply by viceice** [reply]
that issue is fixed, so you probably to update nexus to a newer version
